lib/src/slices/api_app_data/api_app_data_service.dart

```diff
@@ -28,9 +28,8 @@ class ApiAppDataService {
     if (data != null) await _repository.delete(data.id!);
   }
 
-  Future<void> logout() async {
+  Future<void> deleteEmailData() async {
```

Comment:
this logic doesn't belong in this service, this service should just add/update/delete whatever its told. it shouldn't know which keys belong to what. the logic to delete the background fetch keys (label, epoch, page) belong in the data background service.  in the apiAppData service there should be a function called deleteAll which deletes ALL keys on logout